components/unique/Create/CreateToken.vue

```diff
@@ -147,7 +147,7 @@ export default class CreateToken extends mixins(
       data: { collectionEntities },
     } = collections
 
-    this.collections = collectionEntities.nodes?.map((ce: any) => ({
+    this.collections = collectionEntities.nodes?.map((ce) => ({
```

Comment:
Can we have a type?

Reply:
I think I will revert that `any` stuff for now, need more observation each files on `any` types